astor/code_gen.py

```diff
@@ -590,7 +607,13 @@ def recurse(node):
             uni_lit = False  # No formatted byte strings
 
         else:
-            mystr = node.s
+            if isinstance(node, ast.Str):
```

Comment:
You can pass both `node` and `value` to this function. This will simplify the code.

Reply:
@serhiy-storchaka What should I pass in `visit_JoinedStr`? We don't actually use the value in that branch, but I think it's kind of a smell to pass in a throwaway value in that case.

Reply:
Pass None.